docs/process/Estimation-Process.md

```diff
@@ -27,73 +27,130 @@ below the estimate. These assumptions translate to risks - because if
-way. An estimate in time for a task can be calculated from the
-estimate in story points divided by the project velocity, adjusted
-according to staff availability, other factors, etc.
+- **Most-probable estimate (days)**: the number of remaining days of effort the
+  task owner believes he or she will most likely need to finish that task.
 
-The decision on whether a task is included in the next sprint is
-based on its points estimate and the project velocity.
+- **Most-pessimistic estimate (days)**: the number of remaining days of effort
+  the task owner believes he or she will need to be 90% of confident of
+  finishing that task, given a pessimistic appraisal of the situation, and
+  taking into account all of the potential things that could go wrong.
 
-This approach is clearly a **gross simplification of reality**, and so
-we need to be very careful not to produce garbage estimates.
+Note that it's impossible (in general) to predict all the things that could go
+wrong with a task. This is why estimates are estimates, and not commitments.
+Pilots and co-pilots should feel free to give estimates based on their personal
+experience. They may use their accumulated knowledge of how long similar tasks
+have taken in the past to develop new estimates.
 
-### Points
+The **deviation** between the most-probable estimate and most-pessimistic
+estimate allows the authors of each estimate to express their degree of
+uncertainty in the estimate, and it allows people viewing the estimates to gain
+insight into that uncertainty.
 
-| Points | Size                                            |
-| -----: | ----------------------------------------------- |
-|      1 |                                                 |
-|      2 |                                                 |
-|      3 |                                                 |
-|      5 |                                                 |
-|      8 |                                                 |
-|     13 |                                                 |
-|   > 13 | This ticket needs to be split up further.       |
+A very large deviation between the most-probable and most-pessimistic estimate
+tells us that the author is very uncertain in their estimate. We might use
+this information to take action, for example, by breaking up a task into
+smaller, more well-defined pieces.
 
-### Each _Task_ ticket has an estimate
+It's important to realize that:
 
-We assign an estimation to every single task ticket. If there's no
-estimate, then it can't be added to the sprint.
+- These estimates pertain to the number of days of remaining effort required to
+  perform that task **in isolation**. If two or more tasks are performed in
+  parallel, with context switching, then we need to keep the additional cost of
+  that context switching in mind.
 
-### Ticket must be ready to estimate before estimation
+- These estimates refer to a number of days **remaining**, and as such they
```

Comment:
Interesting, will this prove problematic to the current JIRA system? I think it presumes estimates are unchanged from the initial estimate in it's velocity calculations etc.

I don't really find those JIRA features useful anyway, so I don't really care, but might be worth running past Sergiy.